src/app/shared/components/profile-image/profile-image.component.scss

```diff
@@ -0,0 +1,16 @@
+.thumbnail-image{
+    width: 40px;
+    height: 40px;
+}
+h5{
+    margin-top: 10px;
+    font-size: 18px;
```

Comment:
why are we modifying the H5 tag.  we can create  p tag or div tag and add these styles

Reply:
ok